keps/2724-topology-aware-scheduling/README.md

```diff
@@ -257,6 +265,137 @@ the "rack" topology domain (there is only one such Pod, so the requirement
 is satisfied trivially). The PodSet corresponding to the worker only
 prefers the "rack" topology domain, so it may fallback to "block".
 
+###### JobSet with ReplicatedJob instance topology
+
+To address [Story 2](#story-2) we are introducing PodSet Chunks, which are
```

Comment:
Chunk seems really unformal..

Can we call this a PodSet Group? 

Or at least suggest a few names?

Reply:
Thanks for clearing that up. If you and @tenzen-y are happy with chunk than that is fine to me.

Reply:
@mimowo I think `PodSet Slice` might be better since `slice` is a kubernetes term and an infrastructure term like `ResourceSlice` for DRA and Google Cloud TPU Slice. Do you have a good example if `Chunk` would align with Kubernetes term?

Reply:
SGTM, thanks.